src/VS1053.h

```diff
@@ -165,6 +163,8 @@ class VS1053 {
 
     // Clears SCI_DECODE_TIME register (sets 0x00)
     void clearDecodedTime();
+
+    void write_register(uint8_t _reg, uint16_t _value) const;    
```

Comment:
Ok, all we need is to make it public. Interesting that so little change enables loading the latest firmware patches. Very nice.

One thing. All public method have a comments. Are we able to provide a proper description to this method as it is public now?

Reply:
@baldram Another approach would be to integrate the patch in to `ESP_VS1053_Library`.
That would be my preference.
I am willing to code a PR for that.

Reply:
Added the most generic reason.

Reply:
> Added the most generic reason.

Thank you. I was not clear. I rather meant "what" does it do like other comments (not why we do that). Just a short description of write_register and arguments. But example use case is also nice to have in a comment. Can be also added later. Bat maybe no longer relevant according to what you said later :)

What you said (integrate the patch) changes a lot of things. Very interesting. 

Shall I approve and we would merge it for the time being? Or better to merge the full PR?

Not sure I fully see the plan, that would be something about adding a new method like `loadUserCode`.
Will we still need to keep `write_register` method public then? 

> Another approach would be to integrate the patch in to ESP_VS1053_Library.
That would be my preference.
I am willing to code a PR for that.

That sounds very interesting. Eg. the MIDI stuff also requires a patch code. There are more patches like FLAC, etc.
Would be cool if the method for loading user code can be reused later for other patches too, so an argument with plugin could be passed instead of `void`.

Btw. what is a source of patch you'll apply? Is it somewhere here: http://www.vlsi.fi/en/support/software/vs10xxpatches.html 
Any comments regarding the source of the stuff would be very welcome in case of looking for future updates.

Reply:
For now maybe you can just accept and merge this pr. 

I don't have a real plan of action yet, but I feel it should be possible to select which patch(es) are applied.
Maybe open a discussion and flesh out what users want/need and make a plan of action.

Yes the patches come from http://www.vlsi.fi/en/support/software/vs10xxpatches.html
I used v2.9 from http://www.vlsi.fi/fileadmin/software/VS10XX/vs1053b-patches290.zip
From that `vs1053-patches.plg` which I renamed to `vs1053-patches.plg.h`. (So it shows in the Arduino IDE)

Reply:
Here is a link to a discussion: https://github.com/baldram/ESP_VS1053_Library/discussions/66